projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -402,28 +402,32 @@ const VideoPressEdit = CoreVideoEdit =>
 				</Fragment>
 			);
 
-			if ( isUploading ) {
-				return (
-					<Fragment>
-						{ blockSettings }
-						<Loading text={ __( 'Uploading…', 'jetpack' ) } />
-					</Fragment>
-				);
-			}
+			const isFetchingVideo = isFetchingMedia || isFetchingPreview;
+			const displayCoreVideoBlock = ! isUploading && ! isFetchingVideo && ( fallback || ! preview );
```

Comment:
I find this line hard to read, maybe the last bit could be placed in its own const like (`isFallbackOrNotPreview`, or simply placed as the first condition?

`( fallback || ! preview ) && ! isUploading && ! isFetchingVideo`

Reply:
I agree, I moved the if statement that was underneath, I'll see what I can do :)